src/main/sensors/esc_sensor.c

```diff
@@ -3379,6 +3516,10 @@ void escSensorProcess(timeUs_t currentTimeUs)
         DEBUG(ESC_SENSOR_FRAME, DEBUG_FRAME_CRC_ERRORS, totalCrcErrorCount);
         DEBUG(ESC_SENSOR_FRAME, DEBUG_FRAME_TIMEOUTS, totalTimeoutCount);
         DEBUG(ESC_SENSOR_FRAME, DEBUG_FRAME_BUFFER, readBytes);
+#ifdef USE_TELEMETRY_CASTLE
+    } else if (isMotorProtocolCastlePWM()) {
+        castleSensorProcess(currentTimeUs);
+#endif
```

Comment:
Please don't mix up #endif and } block end. It's hard to read

Reply:
Fixed.